sdk/identity/azure-identity/test/ut/managed_identity_credential_test.cpp

```diff
@@ -77,6 +77,18 @@ TEST(ManagedIdentityType, Invalid)
   EXPECT_THROW(
       ManagedIdentityType(ManagedIdentityIdType::SystemAssigned, "clientId"),
       std::invalid_argument);
+
+  EXPECT_THROW(ManagedIdentityType(ManagedIdentityIdType::ClientId, ""), std::invalid_argument);
+  EXPECT_THROW(ManagedIdentityType(ManagedIdentityIdType::ObjectId, ""), std::invalid_argument);
+  EXPECT_THROW(ManagedIdentityType(ManagedIdentityIdType::ResourceId, ""), std::invalid_argument);
+
+  ManagedIdentityCredentialOptions options;
+  options.IdentityType = ManagedIdentityType(static_cast<ManagedIdentityIdType>(99), "");
```

Comment:
Though this isn't a valid value, added to improve code coverage for the `default` case in the switch statement:
![image](https://github.com/user-attachments/assets/0056141c-b0aa-470a-994d-d2272620067c)

Reply:
I hate it when we have to add these checks :). But code coverage ftw :).